sunpy/tests/setup_package.py

```diff
@@ -1,3 +1,3 @@
 def get_package_data():
     return {
-        _ASTROPY_PACKAGE_NAME_ + '.tests': ['coveragerc', 'figure_hashes.json']}
+        _ASTROPY_PACKAGE_NAME_ + '.tests': ['coveragerc', 'figure_hashes_py2.json']}
```

Comment:
because we don't have figure tests for python 3 on 0.7 and I see no real reason to add them.